src/blog/2023/10/citizen-development.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Innovate from within - Why manufacturing must embrace Citizen Developers
+subtitle: Empower your Operational Technology teams as Citizen Developers
+authors: ["marian-demme"]
+#image: blog/2023/10/images/tile-blog-custom-vuetify-components-for-Dashboard.png
+date: 2023-10-23
```

Comment:
@gdziuba @MarianRaphael I think "Build vs Buy" for MES systems should go out tomorrow. Would you be OK with publishing say Wednesday @MarianRaphael ?

Reply:
Sure, it's also okay to wait until next week to have content during our summit.